### ISSUE TITLE
Megadrive 1 - JAP VA4 not working

### ISSUE THREAD

**Issue by VincentNLOBJ** [SukkoPera/MegaDrivePlusPlus#24]
Hi all,

I've acquired a modded MD JAP VA4 with ugly switches and decided to upgrade with MD++. I'm not interested on LED changing colour so I've not touched that part.

Following dragoncity scheme: https://dragoncity17.files.wordpress.com/2015/03/md-jap-va4.jpg:
![image](https://github.com/SukkoPera/MegaDrivePlusPlus/assets/1316870/051d05ab-4c67-4556-a4ca-198c2cc8036c)

My board had JP3 / JP2 traces cut, so I have reconnected them with 2 wires below the board, then cut JP1 / JP4 traces.

After wiring up a this is what I've got:

![image](https://user-images.githubusercontent.com/71412899/134803987-10137831-786e-401f-b0b9-d7439d51ae2d.png)

Arduino powers up, and C button lights up arduino led, which is a good sign, but that's all about it.
Pressing reset button yield no effect (no reset or change language/hz when pressed).

For this reason I'm thinking to have followed a wrong scheme or there is a sort of incompatibility with the reset-in/out on the board.

Anyone have a working scheme of JAP VA4 board?

Cheers
-VincentNL

**Comment by SukkoPera** [comment]
Why did you reconnect JP2/3? All JP1/2/3/4 must be cut!

**Comment by VincentNLOBJ** [comment]
I assumed dragoncity scheme white straight lines meant they had to be connected! but that's an easy one, just removed the connection.

Regarding JP2 left contact seems in a pretty bad shape. 
Multimeter is dead when testing it against all the other JPs, not sure if it's normal, but it's worth to mention.

![1632678391549828127174839733048](https://user-images.githubusercontent.com/71412899/134818396-8eeb06e6-0862-426d-ad2c-31536b043b1c.jpg)

**Comment by SukkoPera** [comment]
Did you cut the trace between Reset In and Out?

Try [enabling debug messages](https://github.com/SukkoPera/MegaDrivePlusPlus/blob/master/MegaDrivePlusPlus.ino#L194) and check what the serial monitor tells you.

**Comment by VincentNLOBJ** [comment]
Yeah I did cut the reset in-out trace, following up dragoncity scheme.
When arduino gets 5v, reset button is disabled so unless it is solved that prevents the mod from working.

I'll try out debugging as you have suggested, thanks!

**Comment by shanotype** [comment]
Hey guys I'm having the same exact issue the reset button doesnt do anything. Any ideas?

**Comment by VincentNLOBJ** [comment]
No, honestly after having made 3 PAL machines working,  JAP VA4 is clearly
not compatible and requires debugging to find out why.

Il Gio 11 Mag 2023, 04:16 shanotype ***@***.***> ha scritto:

**Comment by SukkoPera** [comment]
What does the debugging output say? Can you also check whether the reset button is active-low or not?

**Comment by VincentNLOBJ** [comment]
I'm sorry but I have not access to that unit anymore.

Il Gio 11 Mag 2023, 16:03 Giorgioggì ***@***.***> ha scritto:

**Comment by shanotype** [comment]
I got it working. :)

After looking at PAL VA4 instructions for troubleshooting I noticed it was pretty similar enough and had the trace cut on the reset out end so decided I would join the reset in trace back and cut from reset out side but when removing the wire the whole trace lifted not sure if it got damaged when I soldered the wire to it I should have used thinner wire.

Anyway so I found where the trace lead to and soldered reset out to that, rejoined reset in, and already had a cut from the trace lifting and now everything works.

So to sum it up for people in the future on this model try cutting the same way as PAL VA4 picture. Shame VincentNLOBJ does not have it anymore so we could see if it fixes his same issue. Sorry for the lengthy post guys.

**Comment by SukkoPera** [comment]
This is good news, thanks for reporting back!

**Comment by VincentNLOBJ** [comment]
In case it does not bother you, I'd love to check out pics of the mod you
did. If I'll get the same unit again, will try it out.

Cheers!

Il Gio 11 Mag 2023, 17:59 shanotype ***@***.***> ha scritto:

**Comment by shanotype** [comment]
It's messy atm will probably tidy it up some more. Also noticed your missing that Surface Mount between in and out dunno if that matters.

![20230512_021407](https://github.com/SukkoPera/MegaDrivePlusPlus/assets/58530086/94afdf82-1c5b-4d6d-9a00-8044b5d42ff2)
![20230512_021509](https://github.com/SukkoPera/MegaDrivePlusPlus/assets/58530086/8ab43d06-1072-481e-b090-003f31b644e0)
![20230512_021603](https://github.com/SukkoPera/MegaDrivePlusPlus/assets/58530086/34e1cdc0-729b-47e0-aef1-a11ac75667c5)
![20230512_021739](https://github.com/SukkoPera/MegaDrivePlusPlus/assets/58530086/1345a8e0-d00e-4dbf-b002-a53ce9b472d6)

**Comment by VincentNLOBJ** [comment]
Thanks really appreciate it!

Il Gio 11 Mag 2023, 18:24 shanotype ***@***.***> ha scritto:

**Comment by SukkoPera** [comment]
Link added to Wiki, thanks for your help!
